jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/JKubeServiceHub.java

```diff
@@ -68,62 +68,33 @@
     private LazyBuilder<UndeployService> undeployService;
     private LazyBuilder<MigrateService> migrateService;
     private LazyBuilder<DebugService> debugService;
+    private final boolean connectToCluster;
```

Comment:
Is there any specific reason to call the configuration offline and then the Service Hub variable `connectToCluster`?

If the offline configuration is only used for this purpose and the `connectToCluster` variable is only set by the offline configuration, it would make much more sense to name them the same. It would also help in the future when searching for configurations and debugging.